.github/workflows/maven.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
+
+on:
+  pull_request:
+    branches:
+    - main
+  push:
+    branches:
+    - main
+    - release/*
+  release:
+    types: [created]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up JDK 11
```

Comment:
> The build runs Java 11..primarily these are Java 8 samples and have not necessarily been tested for Java 11 yet, is it possible to build/verify both?

We can add a 2nd step in the actions workflow to build using Java 8. Easy enough, just adding another yml file with different settings.